src/main/java/bio/terra/service/configuration/ConfigurationService.java

```diff
@@ -113,6 +113,17 @@ public ConfigListModel setConfig(ConfigGroupModel groupModel) {
     return new ConfigListModel().items(priorConfigList).total(priorConfigList.size());
   }
 
+  public void setConfigParameterValue(ConfigEnum config, String newValue, String label) {
```

Comment:
It looks like this is only used by test code. I think it can be avoided by mocking the configuration service instead of setting a value in the instantiated copy.

Reply:
See [other comment](https://github.com/DataBiosphere/jade-data-repo/pull/995#discussion_r669970403) - but this is partially addressed by moving this method into testUtils.